package.json

```diff
@@ -5,10 +5,8 @@
   "main": "docs/app/app.module.ts",
   "contributors": [
     "Ashley Hunter",
-    "Alastair McKee",
-    "Pearse McMurray",
     "Donna Marijne Worby",
-    "Chris Brennan"
+    "Tara Todd"
```

Comment:
Ah, could you leave the existing names in place? It's more of a list of people who have contributed significantly to the project, regardless of current status.

Reply:
Re-added